src/main/java/Main.java

```diff
@@ -1,13 +1,16 @@
-
-import org.eclipse.lsp4j.jsonrpc.messages.Either;
-
 import java.io.File;
 
 import magpiebridge.core.MagpieServer;
 import magpiebridge.core.ServerAnalysis;
 import magpiebridge.core.ServerConfiguration;
 import magpiebridge.core.ToolAnalysis;
 
+import analysis.GoblintAnalysis;
+import goblintserver.GoblintClient;
+import goblintserver.GoblintServer;
+
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
```

Comment:
Ooh! Apparently there already is some JSON-RPC library around. Maybe at a later point we should switch to using that then.